packages/patternfly-3/patternfly-react-wooden-tree/src/components/TreeView/TreeView.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info/dist/index';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName } from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+import { TreeView } from './index';
+
```

Comment:
We can either maintain the test data in a separate file as I suggested in [this comment](https://github.com/patternfly/patternfly-react/pull/864/files#r250711598), or we can just hard code it here to use down below